src/pages/edge/adobe-journey-optimizer/in-app-message/tutorials/tabs/javascript-from-native.md

```diff
@@ -1,4 +1,40 @@
-<Variant platform="android" task="obtain" repeat="3"/>
+<Variant platform="android3x" task="obtain" repeat="5"/>
+
+In the `onShow` function of the `PresentationDelegate`, get a reference to the `InAppEventHandler` which can be used for Javascript interactions.
```

Comment:
- `InAppEventHandler `. --> `InAppMessageEventHandler`
-  recommend : "_....., obtain a reference to `InAppMessageEventHandler`for use in Javascript interactions_".